apps/emqx_machine/src/emqx_machine_boot.erl

```diff
@@ -61,6 +61,7 @@ start_autocluster() ->
 stop_apps() ->
     ?SLOG(notice, #{msg => "stopping_emqx_apps"}),
     _ = emqx_alarm_handler:unload(),
+    ok = emqx_app:unset_config_loaded(),
```

Comment:
when new node join cluster, It's will restart apps. 
Before we restart the apps, we should unset `config_loader`. 
otherwise, other nodes will query **this new node is already init config done.**